specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/SubnetCreateWithDelegation.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "subnetName": "subnet1",
+    "virtualNetworkName": "vnetname",
+    "resourceGroupName": "subnet-test",
+    "api-version": "2018-08-01",
+    "subscriptionId": "subId",
+    "subnetParameters": {
+      "properties": {
+        "addressPrefix": "10.0.0.0/16"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/subnet-test/providers/Microsoft.Network/virtualNetworks/vnetname/subnets/subnet1",
+        "name": "subnet1",
+        "properties": {
+          "addressPrefix": "10.0.0.0/16",
+          "provisioningState": "Succeeded",
+          "delegations": [
+            {

```

Comment:
Same here

Reply:
not sure what this comment means

Reply:
According to https://github.com/Azure/azure-rest-api-specs/pull/3693/files#diff-48ae68150e376ac636b1bc2e92b09b32R877, delegation type doesn't have property "id".

Reply:
"id" is a property of every networking resource but I don't see it included in the Swagger spec for most. I was trying to keep parity with the existing Swagger...